contrib/rrdtool/patches/0002-fix-visibility.patch

```diff
@@ -0,0 +1,364 @@
+diff --git a/src/rrd.h b/src/rrd.h
```

Comment:
there's no real point to fix vis with a 400 line patch when you can just not add vis. it doesn't really have any benefit whatsoever here

Reply:
if you want to send it upstream tho then go for it